server/utils/userUtils.ts

```diff
@@ -47,6 +55,9 @@ export const sectionsForUser = (userRoles: Array<string>): Array<ServiceSection>
     items.push(sections.applications)
     items.push(sections.newApplication)
   }
+  if (hasRole(userRoles, 'CAS2_PRISON_BAIL_REFERRER') && config.flags.enablePrisonDashboard) {
```

Comment:
just checking, in the config object enablePrisonDashboard is a boolean true/false, rather than a string 'true'/'false'?